hnn_core/cell_response.py

```diff
@@ -225,13 +226,15 @@ def isoma(self):
     def times(self):
         return self._times
 
-    def reset(self):
+    def reset(self, cell_type_names=None):
```

Comment:
Could `cell_type_names` be an attribute? So you could do `self.cell_type_names` instead of the need to add an argument here?

Reply:
cleaner: https://github.com/jonescompneurolab/hnn-core/pull/357/commits/1b5d24b3fcabdc6cc561d0faccd468294a043079 ?

Reply:
much